recOrder/plugin/workers/acquisition_workers.py

```diff
@@ -878,6 +878,22 @@ def work(self):
 
             self._check_abort()
 
+            # Check that all LF channels have the same exposure settings
+            channel_exposures = []
+            for i in range(len(settings['channels'])):
+                channel_exposures.append(settings['channels'][i]['exposure'])
+
+            channel_exposures = np.array(channel_exposures)
+            if not np.all(channel_exposures == channel_exposures[0]):
+                # TODO: warn user that not all channels exposures were the same
```

Comment:
@talonchandler could you help me with a warning message here? I think a warning on the CLI only is enough, right?

Reply:
I think a warning at the command line is enough. I also need to make sure that the metadata is written correctly, but that should be easy.